examples/config/config.properties

```diff
@@ -0,0 +1,267 @@
+[[periods]]
+name = "ea"
+length = 3
+capacity_factor = 3
+
+[[periods]]
+name = "am"
+length = 4
+capacity_factor = 3.65
+
+[[periods]]
+name = "md"
+length = 5
+capacity_factor = 5
+
+[[periods]]
+name = "pm"
+length = 4
+capacity_factor = 3.65
+
+[[periods]]
+name = "ev"
+length = 8
+capacity_factor = 8
+
+[run]
+air_passenger = true
+non_motorized_skim = true
+highway_skim = true
+transit_skim = true
+global_iterations = 3
+
+[emme]
+num_processors = "max-2"
+
+[emme.scenario_ids]
+ea = 1
+am = 2
+md = 3
+pm = 4
+ev = 5
+
+[emme.highway_maz]
+operating_cost = 18.93
+value_of_time = 17.23
+
+[emme.highway]
+relative_gap = 0.0005
+max_iterations = 30
+tollbooth_start_index = 11
+[[emme.highway.classes]]
+name = "da"
+description = "drive alone"
+emme_mode = "d"
+excluded_links = [ "is_toll_da", "is_sr2",]
```

Comment:
- does this have to be a dummy variable or can it be a combination of requirements?

Reply:
All things are possible, let me know what you have in mind. However, I would like to keep code out of the config and think dummy variables are a good compromise to provide enough flexibility to make simple changes without having to wander into the code base.